packages/lesswrong/lib/types/schemaTypes.ts

```diff
@@ -43,6 +43,7 @@ interface CollectionFieldSpecification<T extends DbObject> {
   getValue?: (doc: T, context: ResolverContext) => any,
   foreignKey?: any,
   logChanges?: boolean,
+  nullable?: boolean,
```

Comment:
Say more about this change? It looks like it's already used in the codebase, so if this wasn't here how come it wasn't type erroring?

Reply:
Not sure, Jim added this

Reply:
@jimrandomh